ui/apps/platform/src/Containers/PolicyCategories/PolicyCategoriesListSection.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { PageSection, Title, Flex } from '@patternfly/react-core';
+
+import PolicyCategoriesList from './PolicyCategoriesList';
+
+type PolicyCategoriesListSectionProps = {
+    policyCategories: string[];
+};
+
+function PolicyCategoriesListSection({ policyCategories }: PolicyCategoriesListSectionProps) {
+    // const [selectedCategory, setSelectedCategory] = useState('');
+    return (
+        <PageSection isFilled id="policy-categories-list-section">
+            <PageSection isFilled variant="light">
```

Comment:
Let’s research an alternative to nested `section` HTML elements to achieve the needed presentation.

Reply:
yeah it's not the best but it was the quickest way to get the look I was looking for. open to other ideas to achieve the same effect that we can adopt across the rest of the UI!